src/subdomains/user/application/repositories/ip-log.repository.ts

```diff
@@ -0,0 +1,5 @@
+import { EntityRepository, Repository } from 'typeorm';
+import { IpLog } from '../../domain/entities/ip-log.entity';
+
+@EntityRepository(IpLog)
+export class IpLogRepository extends Repository<IpLog> {}
```

Comment:
- EntityRespository does not work anymore for LOCK (updated TypeORM, will do the same for DFX)
- Need to add to module?